Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp

```diff
@@ -2241,6 +2241,14 @@ static std::optional<char32_t> pressedCharKey(const String& pressedCharKeyString
 #endif // !ENABLE(WEBDRIVER_KEYBOARD_GRAPHEME_CLUSTERS)
 #endif // ENABLE(WEBDRIVER_ACTIONS_API)
 
+static bool isValidSafeInteger(int value)
+{
+    // Defined in ECMA 2015, §20.1.2.6 Number.MAX_SAFE_INTEGER.
+    // https://www.ecma-international.org/ecma-262/6.0/#sec-number.max_safe_integer
+    int maximumSafeInteger = pow(2.0, 53.0) - 1;
+    return value > maximumSafeInteger * -1 && value <= maximumSafeInteger;
```

Comment:
According to [this,](https://262.ecma-international.org/6.0/#sec-number.min_safe_integer) shouldn't the minimum allowed value be exactly `maximumSafeInteger * -1`, rather than one more than that?
```suggestion
    return value >= maximumSafeInteger * -1 && value <= maximumSafeInteger;
```